server/command/subscribe.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package command
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-msoffice/server/config"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/store"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/utils"
+	"github.com/pkg/errors"
+)
+
+func (h *Handler) subscribe(parameters ...string) (string, error) {
+	user, err := h.UserStore.LoadUser(h.MattermostUserID)
+	if err != nil {
+		return "", err
+	}
+
+	switch {
+	case len(parameters) == 0:
+		client := h.Remote.NewClient(context.Background(), user.OAuth2Token)
+		// sub, err := client.CreateEventSubscription(
```

Comment:
remove comment

Reply:
I am still switching back and forth between different subscription types, figuring out the best way to implement. Will remove later.